### DISCUSSION TITLE
Use wired mouse as wireless mouse

### DISCUSSION THREAD

**Discussion by GigaZelensky** [AppGround-io/bluetooth-keyboard-and-mouse-support#247] [Ideas]
I want to control my laptop wirelessly using my external usb mouse that i have connected to my phone trough a usb hub. i want the mouse to move freely without having to press any button for it to move.
I have been looking for this feature for months now and have not found an app that is cappable of doing this. So if it's possible, please add it.

**Comment by parkerlreed** [comment]
You commented on the other discussion. No need to create a new one. https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/discussions/241#discussioncomment-3181573
